portfolio/src/main/webapp/index.html

```diff
@@ -3,87 +3,77 @@
     <meta charset="UTF-8">
     <title>Justine Ventura</title>
-    <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
   </head>
-  <header>
-      <!--style="color:darkslategrey;
-    font-family:Questrial;
-    font-size:85px;
-    padding-top:20px;
-    text-align=center;
-    padding-left:90px;"-->
-    <div class="center">
+  
+     
+  <body > 
+    <header>
+    
 
-       <h1 class="center" style="color:darkslategrey;margin:auto;width=50%;
-    font-family:Questrial;
-    font-size:85px;
-    padding-top:20px;
-    text-align=center;
-    padding-left:90px;">Justine Ventura</h1>
+    <h1 class="center">Justine Ventura</h1>
    
-    </div>
       <img class="pfp ctr" style="text-align:center" src="img/bitmoji.png"/>
       <br/>
       <div style="margin:auto;width=50%;text-align:center;padding-bottom:15px">
-           <a href="https://www.linkedin.com/in/justine-ventura-6275a8149?originalSubdomain=ca"><i class="fa fa-linkedin fa-3x" 
-      style="color:darkslategrey;
-    border-radius: 50%; margin-left:15px;margin-right:15px" aria-hidden="true"></i></a>
-      <a href="https://github.com/j-vent"><i class="fa fa-github fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px"  aria-hidden="true"></i></a>
-      <a href="https://moma.corp.google.com/person/justineventura"><i class="fa fa-google fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
-    <a href="mailto:justine.ventura14@gmail.com"><i class="fa fa-envelope fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
-    <a href="https://www.instagram.com/venturadventures/?hl=en"><i class="fa fa-instagram fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
+           <a href="https://www.linkedin.com/in/justine-ventura-6275a8149?originalSubdomain=ca"><i class="icon fa fa-linkedin fa-3x" 
+     aria-hidden="true"></i></a>
+      <a href="https://github.com/j-vent"><i class="icon fa fa-github fa-3x"  aria-hidden="true"></i></a>
+      <a href="https://moma.corp.google.com/person/justineventura"><i class="icon fa fa-google fa-3x" aria-hidden="true"></i></a>
+    <a href="mailto:justine.ventura14@gmail.com"><i  class="icon fa fa-envelope fa-3x" aria-hidden="true"></i></a>
```

Comment:
`i` is for *italics* (though it can technically be styled as any kind of emphasis through CSS).

For an Icon element that's set up as a CSS background on another element, you should probably use `span` or `div`. (The main difference between `div` and `span` AFAIK is that `div` defaults to `display: block` and `span` defaults to `display: inline`) https://stackoverflow.com/a/3890309